app/controllers/organizations/pets_controller.rb

```diff
@@ -21,6 +21,7 @@ def edit
   end
 
   def show
+    @active_tab = ["overview", "tasks", "applications", "files"].include?(params[:active_tab]) ? params[:active_tab] : "overview"
```

Comment:
Does "overview" need to be included in the check against the active_tab params since the ternary defaults to it anyway?

Reply:
@aisayo, good point. I will check

Reply:
Thank you change made